product_docs/docs/edbcloud/beta/using_cluster/06_analyze_with_superset.mdx

```diff
@@ -0,0 +1,86 @@
+---
+title: "Analyze with Apache Superset"
+---
+You could use Apache Superset to analyze, explore, and visualize data stored in your PostgreSQL clusters.
+If you are logged in to EDB Cloud and have permissions to access superset, you can open Apache Superset using one of **Analyze** links.
+
+!!! Note
+    Apache Superset shares Single Sign On with EDB Cloud. If you log out of  Superset you will be logged out of EDB Cloud
+
```

Comment:
Although SSO is intuitive here, but can we mention converse case or at-least mention `vice-versa` word please? @skrivov-edb ?